src/authz-module/libraries-manager/components/TeamTable/components/SortDropdown.tsx

```diff
@@ -0,0 +1,96 @@
+import {
+  useContext, useState, useMemo, useCallback,
+  useEffect,
+  FC,
+} from 'react';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  DataTableContext,
+  Dropdown,
+  Icon,
+  Stack,
+} from '@openedx/paragon';
+import { SwapVert } from '@openedx/paragon/icons';
+
+interface SortOption {
+  id: string;
+  desc: boolean;
+  label: string;
+}
+
+interface SortByOptions {
+  [key: string]: Omit<SortOption, 'label'>;
+}
+
+const SORT_BY_OPTIONS: SortByOptions = {
```

Comment:
Not blocking, but it'd be great to update this component to take in the `SortByOptions` and `SORT_LABELS` as props so it can be reused.